browser/tor/tor_profile_manager_browsertest.cc

```diff
@@ -99,8 +102,36 @@ class TorProfileManagerTest : public InProcessBrowserTest {
   MockTorLauncherFactory* GetTorLauncherFactory() {
     return &MockTorLauncherFactory::GetInstance();
   }
+
+  void Relaunch(const base::CommandLine& new_command_line) {
+    base::LaunchProcess(new_command_line, base::LaunchOptionsForTest());
+  }
 };
 
+// We don't run this test on Mac because the function GetCommandLineForRelaunch
+// isn't defined there.
+#if !defined(OS_MAC)
+IN_PROC_BROWSER_TEST_F(TorProfileManagerTest, MAYBE_LaunchWithTorUrl) {
```

Comment:
This will definitely fail on runtime.
```
../../base/test/test_suite.cc:97: Failure
Value of: IsMarkedMaybe(test_info)
  Actual: true
Expected: false
Probably the OS #ifdefs don't include all of the necessary platforms.
Please ensure that no tests have the MAYBE_ prefix after the code is preprocessed.
```

Reply:
Ah, right, I noticed this earlier today. Should be fixed now!